src/upgrade_step_from_4p3p1.py

```diff
@@ -0,0 +1,36 @@
+import os
+
+from src.upgrade_step import UpgradeStep
+
+
+class UpgradeStepFrom4p3p1(UpgradeStep):
+
+    def perform(self, file_access, logger):
+        """
+        Perform the upgrade step from version 0 to 1
+
+        Args:
+            file_access (FileAccess): file access
+            logger (LocalLogger): logger
```

Comment:
The descriptions add nothing. You might as well leave them blank. Admittedly this is the same as other upgrade steps.